rustls-platform-verifier/src/verification/mod.rs

```diff
@@ -77,3 +77,36 @@ fn invalid_certificate(reason: impl Into<String>) -> rustls::Error {
 /// - id-kp-serverAuth
 // TODO: Chromium also allows for `OID_ANY_EKU` on Android.
 pub const ALLOWED_EKUS: &[&str] = &["1.3.6.1.5.5.7.3.1"];
+
+impl Verifier {
+    /// Chainable setter to configure the [`CryptoProvider`][rustls::crypto::CryptoProvider] for this `Verifier`.
+    ///
+    /// This will be used instead of the rustls processs-default `CryptoProvider`, even if one has
+    /// been installed.
+    pub fn with_provider(
+        mut self,
+        crypto_provider: std::sync::Arc<rustls::crypto::CryptoProvider>,
```

Comment:
Nit: Let's import `Arc` instead of fully qualifying it three times in this block.